content/software/systemd-boot-friend/_index.md

```diff
@@ -7,27 +7,42 @@ software_category = "System Administration"
 package_site_url = "https://packages.aosc.io/packages/systemd-boot-friend"
 +++
 
-`systemd-boot-friend` is a script set which helps you to use [systemd-boot](https://www.freedesktop.org/software/systemd/man/systemd-boot.html).
+`systemd-boot-friend` is a software which helps you to use [systemd-boot](https://www.freedesktop.org/software/systemd/man/systemd-boot.html).
 
-There are three scripts in this package:
-- `systemd-boot-friend`, install latest kernels (including mainline and lts kernels) to your EFI partition.
-  - This script will be executed automatically after a kernel modification, so that you will always have the latest kernel in ESP partition.
-  - This script will install `intel-ucode` too.
-- `systemd-boot-mkinit`, generate config files for `systemd-boot`.
-- `systemd-boot-init`, attempt to install `systemd-boot` on your system, try to generate systemd-boot configs, and install the kernels.
+**NOTICE**: If you have installed `systemd-boot-friend`, it will automatically install the newest kernel for you once the kernel is upgraded or modified.
 
-
-**NOTICE**: For now the automatic hook does **NOT** work yet. So you will have to manually run `systemd-boot-friend` after the kernel is updated or modified.
-
-# Usage
+# Initialize
 In order to use `systemd-boot-friend`, you have two options:
-1. Automatically install via using `systemd-boot-init`
+1. Automatically initialize systemd-boot and install the newest kernel via `systemd-boot-friend init`
 2. Manually install and set-up systemd-boot yourself, then utilize `systemd-boot-friend` just to update the kernels.
 
-For the automatic method, simply mount your *ESP partition* to `/efi` and run `systemd-boot-init`. If everything goes well, you should have a working systemd-boot installation.
+For the automatic method, simply mount your *ESP partition* to `/efi` and run `systemd-boot-friend init`. If everything goes well, you should have a working systemd-boot installation.
 
 For the manual method, you can take a look at [systemd-boot - ArchWiki](https://wiki.archlinux.org/index.php/systemd-boot), which describes how to set up systemd-boot by yourself. Then after you have decided on where to mount your *ESP partition*, you can fill in the mountpoint at `/etc/systemd-boot-friend.conf` and you are good to go.
 
+# Usage
+`systemd-boot-friend` has several subcommands.
+
+    init              Initialize systemd-boot-friend
+    mkconf            Create systemd-boot entry config
+    list              List all available kernels
+    install-kernel    Install the specified kernel
+  
+- init
+  Initialize systemd-boot and install the newest kernel.
+
+- mkconf
+  Generates a systemd-boot entry config for the chosen kernel, you can pass custom boot arguments in `/etc/systemd-boot-friend.conf`.
+  If the entry file already exists, it will ask if you want to override the file. You can also pass `-f` or `--force` to force override the file.
```

Comment:
The "force" in "to force override" is unnecessary... I think you mean to say "overwrite," not "override" here. It could also help if you explained how `-f` automatically overwrites the file.